spatialmath/box.go

```diff
@@ -11,15 +11,51 @@ import (
 	"go.viam.com/rdk/utils"
 )
 
-// box density corresponding to how many points per square mm.
-const defaultBoxPointDensity = .5
+// Ordered list of box vertices.
+var boxVertices = [8]r3.Vector{
```

Comment:
Globals like this should generally be the first thing (or one of the first things) in a file after the imports.

I don't want to declare these in the function where they're used, because that's a performance-sensitive function and redeclaring these variables on each call will be noticeably expensive.

Perhaps a different file would be appropriate for these? `geometry_utils.go` if you like

Reply:
Yeah I agree that generally you're right.  I just feel it hurts readability a bit.  You can do whatever you think is right here just wanted to raise this point